packages/notification-center/src/hooks/useFeedUnseenCount.ts

```diff
@@ -10,10 +10,11 @@ export const useFeedUnseenCount = (
   options: UseQueryOptions<ICountData, Error, ICountData> = {}
 ) => {
   const { apiService, isSessionInitialized } = useNovuContext();
+  const unseenQuery = { ...query, seen: query.seen ?? false };
```

Comment:
I understand this was trying to keep retrocompatibility but we can safely ignore the query passed and just set the `unseenQuery` to `seen: false`.
What I would do then is explain in the documentation ([docs/docs/notification-center/react/api-reference.md](https://github.com/novuhq/novu/blob/next/docs/docs/notification-center/react/api-reference.md#usefeedunseencount)) that from earlier versions of `v.0.15.0` (as this would be merged in that version) as deprecated the query needs to be passed but from that version forward is not needed, in the same way as `useUnseenCount` works. Therefore users can slowly clean their code at their pace but withouth the functionality be affected.
@scopsy @unicodeveloper what's your take on this?